package.json

```diff
@@ -1,35 +1,39 @@
 {
-  "name": "airbnb-style",
-  "version": "2.0.0",
+  "name": "eslint-config-dcos",
+  "version": "0.1.0",
   "description": "A mostly reasonable approach to JavaScript.",
-  "scripts": {
-    "preinstall": "npm run install:config && npm run install:config:base",
-    "install:config": "cd packages/eslint-config-airbnb && npm prune && npm install",
-    "install:config:base": "cd packages/eslint-config-airbnb-base && npm prune && npm install",
-    "test": "npm run --silent test:config && npm run --silent test:config:base",
-    "test:config": "cd packages/eslint-config-airbnb; npm test",
-    "test:config:base": "cd packages/eslint-config-airbnb-base; npm test",
-    "travis": "npm run --silent travis:config && npm run --silent travis:config:base",
-    "travis:config": "cd packages/eslint-config-airbnb; npm run travis",
-    "travis:config:base": "cd packages/eslint-config-airbnb-base; npm run travis"
-  },
+  "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/airbnb/javascript.git"
+    "url": "https://github.com/dcos/javascript.git"
   },
   "keywords": [
     "style guide",
     "lint",
-    "airbnb",
     "es6",
     "es2015",
     "react",
     "jsx"
   ],
+  "dependencies": {
+    "eslint-config-airbnb": "^10.0.1"
+  },
+  "peerDependencies": {
+    "eslint": "^3.15.0",
+    "eslint-plugin-compat": "^1.0.2",
+    "eslint-plugin-destructuring": "^2.1.0",
+    "eslint-plugin-import": "^1.13.0",
+    "eslint-plugin-jsx-max-len": "^1.0.0",
+    "eslint-plugin-prettier": "^2.0.1",
+    "prettier": "^0.22.0"
+  },
   "author": "Harrison Shoff <hi@hshoff.com> (https://twitter.com/hshoff)",
```

Comment:
I'm with @mlunoe and think we should eventually remove the author as there is barely any of his work left but would keep him for now.